extension.ts

```diff
@@ -19,15 +19,51 @@ export function activate(context: vscode.ExtensionContext) {
     let projectsStored: string = context.globalState.get<string>('recent', '');
     let aStack: stack.StringStack = new stack.StringStack();
     aStack.fromString(projectsStored);
+    showStatusBar();
     
     // register commands
     vscode.commands.registerCommand('projectManager.saveProject', () => saveProject());
     vscode.commands.registerCommand('projectManager.listProjects', () => listProjects(false));
     vscode.commands.registerCommand('projectManager.listProjectsNewWindow', () => listProjects(true));
     vscode.commands.registerCommand('projectManager.editProjects', () => editProjects());
-    
+
+    function showStatusBar(projectName?: string) {
+
+        let statusItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
```

Comment:
Please add a config/setting to show/hide the status bar. Something like `projectManager.showInStatusBar` (`true` by default). Don't forget to add it in `package.json` as well.